### DISCUSSION TITLE
useFormik initialValues types

### DISCUSSION THREAD

**Discussion by Nurtimax** [Nurtimax/bilimgual#31] [Q&A] [answered]
how to enter types in useFormik into the initial value

**Comment by Matraim** [comment, marked as answer]
interface IFAQValues {
   answer?: string;
   question?: string;
   id?: string;
}

const { setValues, values } = useFormik<IFAQValues[]>({
      initialValues: [{ answer: '', question: '', id: '1' }],
      onSubmit: () => {}
   });
